ui/translations/en-us.yaml

```
@@ -93,3 +93,20 @@ state:
     success: 'Released successfully'
     error: 'Release encountered an error'
     pending: 'Release pending'
+
+status_badge:
+  partial:
+    text: 'Partial'
+    tooltip: 'Application is partially running'
+  down:
+    text: 'Down'
+    tooltip: 'Application is down'
+  ready:
+    text: 'Up'
+    tooltip: 'Application is running'
```

```suggestion
    tooltip: 'Application is ready'
```

Minor nit, but we have both "Ready" and "Alive" states where both are technically "running". So maybe we could be more specific here?